frame/code/ac/FrameUtil.hpp

```diff
@@ -25,6 +25,16 @@ inline std::string_view Frame_getStateChange(const Frame& f) {
     return {};
 }
 
+template<typename StreamOp>
+inline bool Frame_getStreamData(const Frame& f, typename StreamOp::Type& out) {
```

Comment:
this is schema-specific so it's for `schema/FrameHelpers.hpp`

And we already have such functions there: `Frame_to/fromStreamType`